server/migrations/20200518185559-initial-migration.js

```diff
@@ -0,0 +1,21 @@
+const Promise = require('bluebird');
+const fs = require('fs');
+
+module.exports = {
```

Comment:
I like this, and I think it's ok to create our initial state from the flyway dump.